CLAUDE.md

```
@@ -0,0 +1,408 @@
+```
+
+**Memory issues:**
+```bash
+# Increase PHP memory limit in docker-compose.yml
+# Or use ahoy with more memory
+ahoy drush --memory-limit=512M [command]
+```
+
+## Resources
+
+- **Vortex Documentation**: https://vortex.drevops.com
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)